pkgs/tools/audio/beets/common.nix

```diff
@@ -137,14 +138,16 @@ python3Packages.buildPythonApplication {
   ];
 
   nativeCheckInputs =
-    with python3Packages;
     [
+      writableTmpDirAsHomeHook
+    ]
+    ++ (with python3Packages; [
       pytestCheckHook
       pytest-cov
       mock
       rarfile
       responses
-    ]
+    ])
     ++ pluginWrapperBins;
```

Comment:
Nit: The `writeableTmpDirAsHomeHook` addition (nice to know it exists!) can be added here, and the diff might get slightly smaller and nicer :).